### ISSUE TITLE
NVDA screen reader is announcing unnecessary information as "press ALT + F1 key to expand accessibility option" but nothing is happening when we hit these keys:A11y_MonacoEditor_Screenreader

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/monaco-editor#4674]
### GitHubTags:
#A11yTCS; #A11ySev2; #A11yMAS; #WCAG1.3.1; #Desktopweb; #Win11; #Visual Studio Code Client; #VisualStudioCode-Web-Apr2021; #NVDA; #JAWS; #Chrome; #a11y-monacoeditor; #SCIM:bug28133995; #Element:EditField;

### Environment Details:
#URL: [Monaco Editor (microsoft.github.io)](https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0-dev.20230707#XQAAAAJzAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscwzfibW_1uhs0-3YGTBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKimARHSWlrTPOQHROITHxmAetfeRrm3VHdSe7_Etv6WpgzlH4vE8EOD3EPhJDl6tDpOO8RxKZbUytKns3T0NFFloo9Q21KsEBT298I6cyZc5-Glup9UHU6QTpv2Hjab6pZzB3ZO2RH-WGhzXse2-hNlgz6zCgsG8AjiqGg0ZzWzND7z09NmCQOQd6pfkNvO2zmMzFvHpTY8RFEy1urlQcQMb1HHLEESBi8m3Rr3UzFlKzY51hfYcz__QCePc)
Application name: Monaco Editor
Chrome: Version 127.0.6533.73 (Official Build) (64-bit)
Windows 11 Enterprise (Version 23H2)  
OS Build 22631.3737
Screen reader: NVDA 2024.2, JAWS

### Repro Steps:

1. Start NVDA/ JAWS screen reader.
2. Hit the URL: https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0-dev.20230707#XQAAAAJzAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscwzfibW_1uhs0-3YGTBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKimARHSWlrTPOQHROITHxmAetfeRrm3VHdSe7_Etv6WpgzlH4vE8EOD3EPhJDl6tDpOO8RxKZbUytKns3T0NFFloo9Q21KsEBT298I6cyZc5-Glup9UHU6QTpv2Hjab6pZzB3ZO2RH-WGhzXse2-hNlgz6zCgsG8AjiqGg0ZzWzND7z09NmCQOQd6pfkNvO2zmMzFvHpTY8RFEy1urlQcQMb1HHLEESBi8m3Rr3UzFlKzY51hfYcz__QCePc.
3. TAB to Monaco editor.
4. Observe that when we navigate to editor, screen reader is announcing as "press ALT + F1 key to expand accessibility option".

### Actual Results:
NVDA screen reader is announcing unnecessary information as "press ALT + F1 key to expand accessibility option" but nothing is happening when we hit these keys.

**Similar issue is also observed with Chrome + JAWS.**

### Expected Results:
NVDA screen reader should not be announcing unnecessary information as "press ALT + F1 key to expand accessibility option". If these keys are valid then when we hit ALT + F1 key, accessibility options should be available for users.

### User Impact:
Screen reader users will get incorrect information here, as he will try to execute these keys and no action will be performed which may lead to frustration for user.

### Attachment:
[353456995-83df7183-07e4-4ba3-8f3e-2758071c6d6a.webm](https://github.com/user-attachments/assets/7c5f0bc6-f4b5-4c2c-8272-bc9ffb233574)

https://github.com/user-attachments/assets/096c4499-dc4c-46c7-bd2a-496f644a53d7

**Comment by meganrogge** [comment]
this is fixed and can be tested in the preview editor like so

https://github.com/user-attachments/assets/4e2dad97-ea59-4626-b645-1507a7aeb53c

**Comment by chadsowald** [comment]
Thank you @meganrogge!  I can confirm on the preview/RC version that it's fixed.  Do you know when the next stable/formal release will be?

**Comment by meganrogge** [comment]
@hediet would know that

**Comment by PoojaNamde** [comment]
@meganrogge Could you please provide us the latest environment where we can verify the issue.

**Comment by meganrogge** [comment]
you can verify it here https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#example-creating-the-editor-hello-world

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:10-29-24;

Verified the issue in the environment https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#example-creating-the-editor-hello-world. Issue is fixed and not reproducing, hence closing the bug.

![Image](https://github.com/user-attachments/assets/9089eb94-dfcf-453a-b015-a1b29ac0907e)
